.github/workflows/cli.yml

```diff
@@ -87,9 +67,9 @@ jobs:
           path: |
             ~/.cache/go-build
             ~/go/pkg/mod
-          key: ${{ runner.os }}-go-1.19.5-release-cache-${{ hashFiles('cli/go.sum') }}
+          key: ubuntu-go-1.19.5-release-cache-${{ hashFiles('cli/go.sum') }}
           restore-keys: |
-            ${{ runner.os }}-go-1.19.5-release-cache-cli
+            ubuntu-go-1.19.5-release-cache-cli
```

Comment:
Shouldn't this be using `${{ matrix.os }}`?

Reply:
Im not sure this is needed anymore as this step doesn't run on matrix.

Reply:
Ah I see, gotcha